boards/communication/src/data_manager.rs

```diff
@@ -52,74 +34,55 @@ impl DataManager {
         return RadioRate::Slow;
     }
 
-    /// Do not clone instead take to reduce CPU load.
-    pub fn take_sensors(&mut self) -> [Option<Message>; 13] {
-        [
-            self.air.take(),
-            self.ekf_nav_1.take(),
-            self.ekf_nav_2.take(),
-            self.ekf_nav_acc.take(),
-            self.ekf_quat.take(),
-            self.imu_1.take(),
-            self.imu_2.take(),
-            self.utc_time.take(),
-            self.gps_vel.take(),
-            self.gps_vel_acc.take(),
-            self.gps_pos_1.take(),
-            self.gps_pos_2.take(),
-            self.gps_pos_acc.take(),
-        ]
-    }
-
-    pub fn clone_states(&self) -> [Option<StateData>; 1] {
```

Comment:
We should still be sending state messages, and I think we do since we don't treat it differently than any other message received (with the exception of commands).